staging/src/k8s.io/pod-security-admission/policy/check_capabilities_baseline_test.go

```diff
@@ -36,8 +38,34 @@ func TestCapabilitiesBaseline(t *testing.T) {
 					{Name: "a", SecurityContext: &corev1.SecurityContext{Capabilities: &corev1.Capabilities{Add: []corev1.Capability{"FOO", "BAR"}}}},
 					{Name: "b", SecurityContext: &corev1.SecurityContext{Capabilities: &corev1.Capabilities{Add: []corev1.Capability{"BAR", "BAZ"}}}},
 				}}},
-			expectReason: `non-default capabilities`,
-			expectDetail: `containers "a", "b" must not include "BAR", "BAZ", "FOO" in securityContext.capabilities.add`,
+			expectReason:                             `non-default capabilities`,
+			expectDetail:                             `containers "a", "b" must not include "BAR", "BAZ", "FOO" in securityContext.capabilities.add`,
+			enableUserNamespacesPodSecurityStandards: false,
+		},
+		{
+			name: "with hostUsers, but feature gate not enabled",
```

Comment:
should we also test "with hostUsers and feature gate is enabled"

Reply:
added!